src/sentry/models/commit.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+)
+
+
+class Commit(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
+    repository_id = BoundedPositiveIntegerField()
+    key = models.CharField(max_length=64)
```

Comment:
imo it's odd to call this `key` since I've never heard of that nomenclature in any VCS. What about `ref` or `commit`? Also, they're always 40 chars since it's a sha1.